lib/features/search-pad/SearchPad.js

```diff
@@ -521,10 +521,13 @@ function createHtmlText(tokens) {
   var htmlText = '';
 
   tokens.forEach(function(t) {
-    if (t.matched) {
-      htmlText += '<b class="' + SearchPad.RESULT_HIGHLIGHT_CLASS + '">' + escapeHTML(t.matched) + '</b>';
+    var text = escapeHTML(t.matched || t.value);
+    var match = t.match || t.matched;
+
+    if (match) {
+      htmlText += '<b class="' + SearchPad.RESULT_HIGHLIGHT_CLASS + '">' + text + '</b>';
     } else {
-      htmlText += escapeHTML(t.normal);
```

Comment:
so the `normal` property is not used anymore? What was the purpose of it before?

Reply:
Bug fix: https://github.com/bpmn-io/diagram-js/pull/932/commits/32d8f0e0229147f87fb2724371609c4b7cb70906

Test coverage added for rendering (different cases): https://github.com/bpmn-io/diagram-js/pull/932/commits/1fd6df608370e491e90586c95d5553c6faf37335